packages/common/src/api/reducer.ts

```diff
@@ -15,17 +15,17 @@ import { trendingApiReducer } from './trending'
 import { userApiReducer } from './user'
 
 export default combineReducers({
+  accountApi: accountApiReducer,
```

Comment:
is there a way you did this programatically? just curious for my own coding

Reply:
Oh sorting them? You can just sort via the command palette